device/bgp/evpn/evpnmacranges.yaml

```diff
@@ -0,0 +1,46 @@
+components:
+  schemas:
+    Bgp.CMACRanges:
+      description: >-
+        The MAC Ranges per BroadcastDomain
+      type: object
+      required: [mac_addresses]
+      properties:        
+        mac_addresses:
+          description: >-
+           Host MAC address range per Broadcast Domain 
+          $ref: '../../routes/routeaddresses.yaml#/components/schemas/MACRouteAddress'
```

Comment:
should it be ref or include? today we support include.
may be good to support ref.